Makefile

```diff
@@ -1077,7 +1068,13 @@ help:
 	@echo "   test-profiling run pillar tests with memory profiler"
 	@echo "   clean          clean build artifacts in a current directory (doesn't clean Docker)"
 	@echo "   release        prepare branch for a release (VERSION=x.y.z required)"
-	@echo "   patch          make a patch release on a current branch (must be a release branch)"
+	@echo "   rc-release     make a rc release on a current branch (must be a release branch)" 
```

Comment:
yetus is reporting two cases of blanks at the end of line.

Reply:
Fixed now. Had to park this PR for other priorities.